web-app/HeimScripts/heatmap/summary.R

```diff
@@ -15,8 +15,8 @@
 #     are not overwritten if the script is run for the 'preprocess data' tab)
```

Comment:
You do not make use of "preprocessed" variable right? Is it intended? How will you generate summary stats for the preprocessed data?